<?xml version="1.0"?>
<launch>

  <node pkg="cob_relayboard" type="cob_relayboard_node" name="relayboard" respawn="false" output="screen">
	<param name="ComPort" value="/dev/ttyUSB0"/>
  </node>

</launch>
